lib/tests/s3-upload.test.js

```diff
@@ -10,9 +10,12 @@ jest.mock("uuid", () => {
 describe("S3Client", () => {
   it("Should raise an error with malformed parameters", async () => {
     jest.spyOn(global, "Date").mockImplementationOnce(() => new Date("2021-08-12"));
-    await uploadFileToS3(Buffer.from("test"), "temp-s3-upload-testing", "test").catch((error) => {
-      expect(error).toBeTruthy();
-    });
+    expect(
```

Comment:
Not your test but I refactored because it was throwing a fail locally.  It now tests specifically for the TypeError.